src/styles/sidebar/components/annotation-header.scss

```diff
@@ -13,6 +13,15 @@
     align-items: baseline;
   }
 
+  &__reply-toggle.button {
```

Comment:
I noticed the hover color is no longer red, I'm not sure I like the red, but if we're gonna match up with the old button we may need to add that back too or was that intentional? I think we may need to have only red hover be for real links correct?

Reply:
Good catch. I've added back in the red for now. One day when this hectic migration is over, there's gonna be a reckoning of this stuff!